pom.xml

```diff
@@ -83,10 +83,8 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.17</version>
+        <version>${maven.surefire.plugin.version}</version>
```

Comment:
Not necessary for this fix but I upgraded the version as part of debugging it so we might as well stick with latest and greatest since it works.